### DISCUSSION TITLE
BIRT export to word - table borders do not appear

### DISCUSSION THREAD

**Discussion by stevehydro** [eclipse-birt/birt#1528] [Q&A]
Hi,

When I export a report to Word that has table borders, the borders do not appear in the Word export.  They do appear if I use a grid though. 

Any ideas as to why?

Version: Neon.3 Release (4.6.3)

Thanks!

**Comment by hvbtup** [comment]
That might be a bug.
You can try using cell borders instead of table or row borders.

**Reply by stevehydro** [reply]
I had to insert a Grid into the areas where I wanted borders, lots of extra work for such a silly thing.

**Reply by speckyspooky** [reply]
I have created a very small report based on the BIRT-classic-car-models with BIRT 4.6.
The result is that the borders of my table will be displayed correctly in DOCX - result see below.
I have added also my test-report and the DOCX below.

For me it seems to be a special constellation on your report. We haven't any details about your report.
- Have you used a special border styling of the border?
- Is it a small or complex report?
- etc.

I cannot reproduce a border error with BIRT 4.6. If I would able to reproduce the topic
then may be we can assist you and we could check the latest version too (to verify the current system behavior).


### BIRT 4.6 Table border to DOCX
![2023-12-21 23_41_56-word_border_report(2) docx  -  Kompatibilitätsmodus - Word](https://github.com/eclipse-birt/birt/assets/41593722/2326f6c9-228a-4205-b415-3052670d6c72)

### Test-report and DOCX
[word_border_report.docx](https://github.com/eclipse-birt/birt/files/13746668/word_border_report.docx)
[word_border_report.zip](https://github.com/eclipse-birt/birt/files/13746669/word_border_report.zip)
